CommonData/column.cpp

```diff
@@ -65,12 +65,22 @@ void Column::loadComputedColumnJsonBackwardsCompatibly(const Json::Value & json)
 	//Columnname is used to find this:
 	//_column = DataSetPackage::pkg()->dataSet()->column(json["name"].asString());
 	
-	const std::string & rCode = json["rCode"].asString();
-	
+	const std::string & rCode = json["rCode"].asString(),
+					  & codeTypeStr = json["codeType"].asString();
+
+	computedColumnType codeType = computedColumnType::notComputed;
+	if (!codeTypeStr.empty())
+	{
+		try { codeType = computedColumnTypeFromString(codeTypeStr); }
+		catch(...) {}
+	}
+
+	try { codeType = computedColumnTypeFromString(codeTypeStr); } catch(...) {}
+
 	setCompColStuff
 	(
 		json["invalidated"].asBool(),
-		computedColumnTypeFromString(json["codeType"].asString()),
+		codeType,
 		rCode,
```

Comment:
You dont need to make changes to column.cpp I think?

Just the change to `DatabaseInterface::columnGetComputedInfo` should be enough

Reply:
I was not quite completely sure that this error could not arise here. For the case, it does not hurt I fimd.

Reply:
Yeah but if the changes are not necessary at all why make them?